api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/GraphProcess.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import io.circe.parser.parse
+import io.circe.{Decoder, Encoder, Json}
+
+object GraphProcess {
+
+  implicit val encoder: Encoder[GraphProcess] = Encoder.encodeJson.contramap(_.json)
+  implicit val decoder: Decoder[GraphProcess] = Decoder.decodeJson.map(GraphProcess(_))
+
+  val empty: GraphProcess = GraphProcess("{}")
+
+  //TODO: Add json validation here?
+  def apply(jsonString: String): GraphProcess = {
```

Comment:
You can just use CirceUtil.decode here if you defined codec...

Reply:
I saw decoding in another place - try to decode json to Canonical form.. But maybe in next step?